README.md

```diff
@@ -184,106 +208,8 @@ workon taiga
 pip install -e .
 ```
 
-Modify `taiga-back/settings/local.py` and include the line:
+Modify `taiga-back/settings/local.py`. See [Taiga Back](#Taiga Back) section for content:
```

Comment:
Excellent idea, but unfortunately that link does not seem to work.
I would suggest writing `See "Production env -> Taiga Back" section for content.`